apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 package v1beta2
 
 import (
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
```

Comment:
we should not have coupling with the FLP api here, the config definitions should be duplicated (or if we force the semantic convention then we can just remove that)

Reply:
I duplicated the `GenericTransformRule`